examples/01_plotting/plot_3d_map_projection.py

```diff
@@ -0,0 +1,38 @@
+"""
```

Comment:
I would write filename as plot_3d_map_to_surface_projection.py ?

Reply:
General comment: Is there any easy way to compare between volume and surface projection of same statistical map ?

Reply:
should we plot both the surface projection and some slices or a glass brain in this example?

Reply:
I don't know much about surface plotting. I am justing trying to visualize both projections and thinking how would they look like. May be a general comment.

Reply:
added a glass brain to compare